Gaijinizer patch/Map407.txt

```diff
@@ -4,404 +4,518 @@
+> CONTEXT: Map407/events/1/pages/0/169/Dialogue/enemy29
+Haaah... what a mood killer...
 
+Fine, it's my loss.#Ron
 > END STRING
 
 > BEGIN STRING
 最初が、あんなに意味ありげに登場したから
 最後くらい、強キャラとして華々しく戦って…………
 
 印象深く退場しようと思ったんだけどなぁ
-> CONTEXT: Map407/events/1/pages/0/173/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map407/events/1/pages/0/173/Dialogue/enemy29
+At first I thought there was some meaning to your
+appearance here, and we'd have some spectacular
+battle at the end. And then I'd have some really
+memorable exit from the scene.#Ron
 > END STRING
 
 > BEGIN STRING
 ………………ボクはね
 最初は、ホントにキミを始末するつもりだった。
 
 この身体も、そのためだけに実体化したんだ。
-> CONTEXT: Map407/events/1/pages/0/183/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map407/events/1/pages/0/183/Dialogue/enemy29
+The thing is, I really did intend on disposing of
+you at the start. That's the reason I changed my
+body into this form.#Ron
 > END STRING
 
 > BEGIN STRING
 でもやっばり、これから抱いて殺す相手なんだから
 ちゃんと知った上で、好意で抱きたいじゃない？
 
 だから、キミを見定めるつもりだったんだ。
-> CONTEXT: Map407/events/1/pages/0/188/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map407/events/1/pages/0/188/Dialogue/enemy29
+But of course, you want to know about the person
+you're going to embrace to death, as a token of
+goodwill to them. That's why I've been trying to
+get a handle on what kind of person you are.#Ron
 > END STRING
 
 > BEGIN STRING
 けどそれは、最初のキミの攻撃でもう『答え』が出てた。
 
 キミの戦いには、『殺気』がない
 そりゃあもう、ドン引きするぐらいにね。
-> CONTEXT: Map407/events/1/pages/0/193/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map407/events/1/pages/0/193/Dialogue/enemy29
+But I had my "answer" when I first attacked you.
+There was no killing intent when you fought.
 
+It left me pretty shocked.#Ron
 > END STRING
 
 > BEGIN STRING
 確かに、相手を憎まず、傷つけることなく
 お互いに無事で済ませるのは、とても理想的だ。
 
 でも、生死をかけた争いの中では、そうはいかない。
-> CONTEXT: Map407/events/1/pages/0/198/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map407/events/1/pages/0/198/Dialogue/enemy29
+Of course, it would be ideal if battles could be
+concluded with everyone walking away safe and
+sound. But that's not how it works in
+life-and-death struggles.#Ron
 > END STRING
 
 > BEGIN STRING
 思わずイラッとしたり、カチンときたり
 相手の実力によっては、傷付けない加減が難しかったり
 
 殺意っていうのは、ちょっとしたことで沸いちゃう
-> CONTEXT: Map407/events/1/pages/0/203/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map407/events/1/pages/0/203/Dialogue/enemy29
+You can find yourself irritated and annoyed,
+and depending on your opponent's strength, it can
+be difficult to avoid hurting them. Any old thing
+can be enough to arouse the desire to kill.#Ron
 > END STRING
 
 > BEGIN STRING
 特にキミの場合、『天使の魔力』の特性上
 ほんのすこしでも間違いは許されないじゃない？
 
 だから、始末したほうが手っ取り早いと思ってたんだ。
-> CONTEXT: Map407/events/1/pages/0/208/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map407/events/1/pages/0/208/Dialogue/enemy29
+And particularly in your case with that "angelic
+magic" you've got. You can't afford even the
+slightest misstep. So that's why I thought
+disposing of you would be the fastest solution.#Ron
 > END STRING
```

Comment:
```
In your case specifically, you can't afford even
the slightest misstep with that "Angelic Magic"
you've got. So that's why I thought disposing of
you would be the fastest solution.
```